.gitignore

```
@@ -133,3 +133,6 @@ dmypy.json
 
 # vscode
 .vscode/
+notebooks/TL_v1/variables/variables.data-00000-of-00001
+.gitattributes
+notebooks/TL_v2/variables/variables.data-00000-of-00001
```

```suggestion
notebooks/TL_v1/
.gitattributes
notebooks/TL_v2/
```
This gets at what you were doing without having to specify file names (in case they change).  Alternatively, you could move saved (large) models to a single folder and exclude that entire folder